backend/backend.go

```diff
@@ -78,16 +126,16 @@ func (backend *Backend) ProcessCENReport(cenReport *CENReport) (err error) {
 	// store the cenreportID in cenkeys table, one row per key
 	for _, cenKey := range cenKeys {
 		cenKey := strings.Trim(cenKey, " \n")
-		if len(cenKey) > 30 && len(cenKey) <= 32 {
-			_, err = stmtKeys.Exec(cenKey, reportID, curTS)
```

Comment:
Could we get a new unit test in `backend_test.go` that demonstrates that the timestamp used is in fact the one coming from the report?

Reply:
Fixed in main code doing the generation of sample data...